### ISSUE TITLE
Add a button to navigate to the previous question.

### ISSUE THREAD

**Issue by SriramPatibanda** [dscnitrourkela/project-maple#62]
**Is your feature request related to a problem? Please describe.**
Yes. Currently, there's no button to go to the previous question directly. One would've to select the question from the question selector dropdown.

**Describe the solution you'd like**
Create a button that displays the previous question when pressed.

**Describe alternatives you've considered**
N.A.

**Additional context**
N.A.

**Comment by SoulPlayer23** [comment]
Can you show the current UI to get an idea of the issue?

**Comment by SriramPatibanda** [comment]
> Can you show the current UI to get an idea of the issue?

Hey @SoulPlayer23 , 
I appreciate your interest in our app. Please feel free to setup the project on your local environment and explore the app to get an idea about the current UI.

**Comment by nayakastha** [comment]
Hello @SoulPlayer23,  check [this]( https://www.figma.com/file/Wkh50Ayhok3ZszMqqCnF7u/DSC-X-AIChE-App?node-id=0%3A1) out  for UI of Aptiche

**Comment by SoulPlayer23** [comment]
Shall I give it a try? Is there any google services I should look out for?

**Comment by nayakastha** [comment]
Yes sure! We would love to have you as our contributor. But could you give us a day or two? The first release of the application is scheduled and we can continue after that.

**Comment by SoulPlayer23** [comment]
Sure. I just checked the files and it looked like it was missing the google service json file. I'll look into the issue after your release.
